frontend/www/app/app/menu.html

```diff
@@ -23,7 +23,9 @@ <h1 class="title">{{currentUser.profile.name? currentUser.profile.name : current
         <ion-item nav-clear menu-close href="#/app/actions/yours">
           <span translate>Actions</span>
         </ion-item>
-        <ion-item nav-clear menu-close ng-show="currentUser.householdId&&households[currentUser.householdId].connected===true && !currentUser.testbed.isStockholm() " href="#/app/prosumption">
+        <!--  <ion-item nav-clear menu-close ng-show="currentUser.householdId&&households[currentUser.householdId].connected===true && !currentUser.testbed.isStockholm() " href="#/app/prosumption"> -->  
+        <ion-item nav-clear menu-close ng-show="!((currentUser.profile.testLocation=='') || (currentUser.profile.contractId==''))" href="#/app/prosumption">  
```

Comment:
This tab is shown to Stockholm users (probably because their testLocation and contractId are not set at all, so they are undefined). Consider using negative version of the condition that shows the Stockholm only tabs (the ones for cooperative).